app/redux/ui/actions.ts

```diff
@@ -14,7 +14,11 @@ export const setOsTheme = () => async (dispatch: AppThDispatch) => {
   dispatch({ type: SET_OS_THEME, payload: { isDarkTheme } });
 };
 
-export const switchTheme = () => ({ type: THEME_SWITCHER });
+export const switchTheme = () => async (dispatch: AppThDispatch, getState: GetState) => {
```

Comment:
what is the reason to this change?
`eventsService.switchTheme` should appear next to `switchTheme` call in Settings screen

Reply:
Fixed